h/static/scripts/guest.coffee

```diff
@@ -407,13 +392,11 @@ module.exports = class Annotator.Guest extends Annotator
     event.preventDefault()
     event.stopPropagation()
     @adder.hide()
-    annotation = this.setupAnnotation(this.createAnnotation())
+    # Check to see if the highlight or note button was pushed.
+    if event.target.className == "h-icon-border-color annotator-highlighter"
```

Comment:
Brittle. I'm changing this to `$(event.target).hasClass('annotator-highlighter')`. The icon is of no concern.